internal/bundle/plain_test.go

```diff
@@ -0,0 +1,121 @@
+package bundle
+
+import (
+	"io/fs"
+	"testing/fstest"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Plain bundle v0 loader", func() {
```

Comment:
Nit: pretty subjective, but when a Describe is clearly scoped to a particular function or type, I typically use the function name as the describe string.
```go
var _ = Describe("LoadPlainV0", func() { ... })
```